ops/production-deploy.tmpl.yaml

```diff
@@ -0,0 +1,580 @@
+replicaCount: 2
+
+resources:
+  limits:
+    memory: "8Gi"
```

Comment:
@orangewolf should the limits match what's currently in [prod](https://github.com/notch8/palni-palci/blob/main/ops/production-deploy.tmpl.yaml#L6-L12)?

Reply:
https://github.com/notch8/palni_palci_knapsack/blob/main/ops/staging-deploy.tmpl.yaml#L3-L8